### PR TITLE
new feature: specifiy --path multiple times (no API change)

### PR DESCRIPTION
**Pull request by OlliL**
As an alternative to #32 without changing the API

### CODE REVIEW

File: bosch_thermostat_client/bosch_cli.py [hunk 1]
```diff
@@ -76,7 +76,9 @@ async def _scan(gateway, smallscan, output, stdout):
 
 async def _runquery(gateway, path):
     _LOGGER.debug("Trying to connect to gateway.")
-    result = await gateway.raw_query(path)
+    result = []
+    for p in path:
+        result.append(await gateway.raw_query(p))
```
Comment (pszafer) [inline bosch_thermostat_client/bosch_cli.py:81]: I'd add `await asyncio.sleep(0.3)` just to be sure that one query after another won't lock you from API.

Comment (OlliL) [reply bosch_thermostat_client/bosch_cli.py:81]: I didn't found a similar sleep in helper.py#deep_into thats why I didn't add smth like that but maybe I missed something. Anyway - you are far more experienced with that code and those devices :smile:. I added the sleep.

**Comment by OlliL** [comment]
Anything I could possibly do about those failed checks? Looks like a problem originating from the checked in .pdm.toml and the github build cluster which just don't match.

**Comment by pszafer** [comment]
No, it's my mistake. I will fix that

File: bosch_thermostat_client/bosch_cli.py (hunk 1 above)
Comment (pszafer) [reply bosch_thermostat_client/bosch_cli.py:81]: I assume that you'd use this method often so better for your device. Raw scan is only for debug purposes so I didn't care that it doesn't have any sleep ;)

**Pull request merged**